doc/configuration.rst

```diff
@@ -681,6 +681,21 @@ Port to listen to
 
 
 
+.. _MANAGEMENT_PROXY_URL:
```

Comment:
Side note, if I'm not wrong, this will be lost. This file `doc/configuration.rst` is autogenerated from the `pyqgisserver/config.yml`

Reply:
Thanks. The PR has been updated. `config.yml` is now used.